pkg/api/types.go

```diff
@@ -370,6 +370,9 @@ type PersistentVolumeSource struct {
 	// ScaleIO represents a ScaleIO persistent volume attached and mounted on Kubernetes nodes.
 	// +optional
 	ScaleIO *ScaleIOVolumeSource
+	// LocalVolume represents a local volume accessible only from one node
+	// +optional
+	LocalVolume *LocalVolumeSource
```

Comment:
I thought that just "Local" would be too generic.  But I don't feel strongly about either naming option, so am totally open changing it.